KDICCore/KDICPlayer.swift

```diff
@@ -3,25 +3,28 @@ import AVFoundation
 
 private let streamURL = URL(string: "http://kdic.grinnell.edu/stream")!
 
+private(set) var isPlaying = false;
+
 open class KDICPlayer {
   private static var player: AVPlayer = {
     let asset = AVURLAsset(url: streamURL)
     let playerItem = AVPlayerItem(asset: asset)
+    let currentlyPlaying = isPlaying;
     return AVPlayer(playerItem: playerItem)
   }()
   
-  open static var isPlaying = player.rate != 0
-  
   open class func play() {
     player.play()
+    isPlaying = true
   }
   
   open class func pause() {
     player.pause()
+    isPlaying = false
   }
   
   open class func toggle() {
-    if isPlaying {
+    if isPlaying == true {
```

Comment:
Do we need to have `== true` here? Please remove if it compiles without it. `isPlaying` is a `Bool`.